### ISSUE TITLE
Example filter has incorrect parameters 

### ISSUE THREAD

**Issue by Craggle** [MicrosoftDocs/dynamics365smb-devitpro-pb#683]
Hiya, the example for substringof	"filter=substringof(Name, ‘urn’)" has the input parameters in the wrong order.  When testing in practice,  "filter=substringof(‘urn’, Name)" is the only pattern that returns the expected results. 

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 090c9021-b9ba-59af-7dea-20f998cec823
* Version Independent ID: 0ff459c1-2620-38e5-8765-20cacf8b37f9
* Content: [Using Filter Expressions in OData URIs - Dynamics NAV](https://docs.microsoft.com/en-us/dynamics-nav/using-filter-expressions-in-odata-uris#feedback)
* Content Source: [dynamics-nav/Using-Filter-Expressions-in-OData-URIs.md](https://github.com/MicrosoftDocs/navdevitpro-content-pr/blob/live/dynamics-nav/Using-Filter-Expressions-in-OData-URIs.md)
* Product: **dynamics-nav-2018**
* GitHub Login: @edupont04
* Microsoft Alias: **solsen**

**Comment by learn-by-flying** [comment]
I found the same behavior but can confirm substringof works for V3 of odata whereas v4 mandates the use of contains().

[‎9/‎13/‎2019 8:10 AM]  
No, it searches via /Customers but if you use /ODATAv4/ you cannot use the substringof method and must you contains.
https://stackoverflow.com/questions/24994774/webapi-2-2-does-not-support-substringof-function 
Found the answer here.
Also noticed that Contains(Name,'Body') returns different search results as contains(Search_Name,'Body')
